robyn/cli.py

```diff
@@ -90,14 +90,10 @@ def start_dev_server(config: Config, file_path: Optional[str] = None):
 
 
 def start_app_normally(config: Config):
-    # Parsing the known and unknown arguments
-    known_arguments, unknown_args = config.parser.parse_known_args()
+    command = [sys.executable]
 
-    # Convert known arguments to a list of strings suitable for subprocess.run
-    known_args_list = [f"{key}={value}" for key, value in vars(known_arguments).items() if value is not None]
-
-    # Combine the python executable, unknown arguments, and known arguments
-    command = [sys.executable, *unknown_args, *known_args_list]
```

Comment:
@sansyrox please enlighten me on why there was this much jargon here...? 🤔 it would suffice to pass argv values directly to the subprocess, since we are def doing the argument parsing inside `Robyn#init`